ironweb.d.ts

```diff
@@ -138,6 +139,10 @@ export interface DeviceKeys {
     signingPrivateKey: PrivateKey<Base64String>;
 }
 
+export interface UserCreateOptions {
+    needsRotation?: boolean;
+}
```

Comment:
You need to add this as the type of a new optional argument to the `createNewUser` method down below on line 276.